pkgpanda/util.py

```diff
@@ -160,6 +160,19 @@ def islink(path):
         return os.path.islink(path)
 
 
+def realpath(path):
```

Comment:
If `islink` is modified to return `False` for all Windows files, then this function can simply be replaced with `os.path.realpath`.

Reply:
See https://github.com/dcos/dcos/pull/3473#discussion_r251722959, I don't think we can get away with always returning `False` on Windows.